armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -833,15 +862,20 @@ def setUp(self):
         )
         reduceTestReactorRings(r, o.cs, 3)
 
-        self.stdAssems = [a for a in r.core.getAssemblies()]
+        self.stdAssems = list(a for a in r.core.getAssemblies())
 
         oCold, rCold = loadTestReactor(
             os.path.join(TEST_ROOT, "detailedAxialExpansion"),
             customSettings={"inputHeightsConsideredHot": False},
         )
         reduceTestReactorRings(rCold, oCold.cs, 3)
 
-        self.testAssems = [a for a in rCold.core.getAssemblies()]
+        self.testAssems = list(a for a in rCold.core.getAssemblies())
+
+    def tearDown(self):
```

Comment:
I'm not sure this would help.

I've been thinking about it. But imagine we have 4 unit tests running at the same time. Some are axial expansion, some are not.

If you set and reset the (stupid, annoying) global CS object in one of these tests, it could screw up the other test.  I'm not sure it matters if you set or unset.

This change might lead to fewer failures because a random unit test is running at the same time as an axial expansion test, and this might save the day. But imagine if two axial expansion tests are running at the same time; then this reset would BREAK a unit test.

I haven't ignored this PR. I'm just thinking.

Reply:
Ah ok, I think I better understand the problem now. 

I agree, this PR will likely not 100% fix the problem. Darn. I'll revert it to a draft and re-open when I get a new solution.

Reply:
I think this problem will go away when we remove the `getMasterCs()` function, finally.  That will remove it from `loadTestReactor()`, and ARMI should be A LOT more parallelizable.

Unfortunately, that's been bumped for a while, due to releases and material things.

Reply:
@john-science should we plan on addressing #983 via the removal of `getMasterCs()` and `setMasterCs()` then? And more-or-less scrap this PR?